certified-connectors/Charles Universe/README.md

```diff
@@ -0,0 +1,13 @@
+## Charles Universe
```

Comment:
@eljefedelrodeodeljefe Please provide a detailed description of your connector which anyone can understand.

Reply:
@eljefedelrodeodeljefe , you can refer to the readme.md [here](https://github.com/st0014/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md). readme.md file is only required for open sourcing your connector which can be helpful to the open source users who may want to contribute in your connector.